spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/catalog/TestTransformTableChange.java

```diff
@@ -35,4 +38,46 @@ void testTransformRemoveProperty() {
         (com.datastrato.gravitino.rel.TableChange.RemoveProperty) tableChange;
     Assertions.assertEquals("key", gravitinoRemoveProperty.getProperty());
   }
+
+  @Test
+  void testTransformAddColumn() {
+    TableChange sparkChange = TableChange.addColumn(new String[] {"col1"}, DataTypes.StringType);
+    com.datastrato.gravitino.rel.TableChange gravitinoChange =
+        GravitinoCatalog.transformTableChange(sparkChange);
+
+    TableChange.AddColumn sparkAddColumn = (TableChange.AddColumn) sparkChange;
+    Assertions.assertTrue(
+        gravitinoChange instanceof com.datastrato.gravitino.rel.TableChange.AddColumn);
+    com.datastrato.gravitino.rel.TableChange.AddColumn gravitinoAddColumn =
+        (com.datastrato.gravitino.rel.TableChange.AddColumn) gravitinoChange;
+
+    Assertions.assertEquals(1, sparkAddColumn.fieldNames().length);
+    Assertions.assertEquals(1, gravitinoAddColumn.fieldName().length);
+    Assertions.assertEquals(
```

Comment:
could simple the code like  Assertions.assertEquals(sparkAddColumn.fieldNames(), gravitinoAddColumn.fieldName())